pkg/credentialprovider/apis/criauth/register.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package criauth
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "criauth.k8s.io"
```

Comment:
> Do we consider the CRIAuthPluginConfig type tied to the kubelet?

I think this is a fair assumption but I would think `CredentialProviderConfig` and `CredentialProviderRequest` should be in the same API group. Are you suggesting we move `CredentialProviderConfig` to `k8s.io/kubelet` with the group name `kubelet.config.k8s.io` and keep `CredentialProviderRequest` / `CredentialProviderResponse` with a group name `credentialprovider.authentication.k8s.io/v1alpha1` here for now?

I initially stayed away from naming things `CredentialProvider*` since I didn't want to overlap too much with other efforts like the client-go exec plugins. Also figured it would be useful to be more specific on what the credentials are for.

Reply:
talked offline, settled on

```
kind: CredentialProviderRequest
apiVersion: credentialprovider.kubelet.k8s.io/v1alpha1
```